vllm/v1/core/sched/scheduler.py

```diff
@@ -557,7 +557,10 @@ def schedule(self) -> SchedulerOutput:
 
         events = self.kv_cache_manager.take_events()
         if events:
-            batch = KVEventBatch(ts=time.time(), events=events)
+            dp_rank = self.vllm_config.parallel_config.data_parallel_rank
```

Comment:
Could it be an option for the event publisher to set the dp rank in each message since it's initialized with `parallel_config` anyhow, to keep the logic here slightly simpler? (no big deal if not)